docs/sql-ref-syntax-aux-show-table.md

```diff
@@ -50,10 +51,14 @@ SHOW TABLE EXTENDED [IN|FROM database_name] LIKE 'identifier_with_wildcards' [PA
        <li> The leading and trailing blanks are trimmed in the input pattern before processing.</li>
     </ul> 
   </dd>
-  <dt><code><em>PARTITION(partition_spec)</em></code></dt>
+  <dt><code><em>partition_spec</em></code></dt>
   <dd>
-    Specifies partition column and its value which is exists in the table. Note that a table regex 
-    cannot be used with a partition specification..
+    An optional parameter that specifies a comma separated list of key and value pairs
```

Comment:
@huaxingao here there is a mention about mutual exclusivity between the LIKE pattern and partition spec. Can we mention it here as well ?